### PR TITLE
Flake8 linting 

### PR DESCRIPTION
**Pull request by joachimwolff**
Flake8 linting with autopep8 without the rules E501,F403,E402,F999,F405. If I changed something manually I added a comment starting with 'wolffj'.

### CODE REVIEW

File: galaxy/wrapper/tool_test_output.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+|  tool error
+| Dataset Info:
+|  Conda dependency seemingly installed but failed to build job environment.
+| Dataset Job Standard Output:
+|  *Standard output was empty.*
+| Dataset Job Standard Error:
+|  *Standard error was empty.*
+|
+--------------------------------------
+</pre></div>
```
Comment (bgruening) [inline galaxy/wrapper/tool_test_output.html]: This file should not be here.

File: hicexplorer/hicBuildMatrix.py [hunk 2]
```diff
@@ -337,21 +337,22 @@ def get_rf_bins(rf_cut_intervals, min_distance=200, max_distance=800):
...
         new_chrom.append(chrom[idx])
 
     new_end.append(end[-1])
     assert len(new_chrom) == len(new_start), "error"
     assert len(new_end) == len(new_start), "error"
 
     intervals = zip(new_chrom, new_start, new_end)
-    intervals = [(chrom, start, end) for chrom, start, end in intervals if end - start >= min_distance]
+    # wolffj: Redefinition: variables 'chrom', 'start' and 'end' are already defined in line 313.
+    intervals = [(chrom_, start_, end_) for chrom_, start_, end_ in intervals if end_ - start_ >= min_distance]
```
Comment (bgruening) [inline hicexplorer/hicBuildMatrix.py]: normally you start these variables with `_` not end them.

**Comment by bgruening** [comment]
Looks good to me. Can you remove your `wolffj` comments and remove the wrongly committed files. Then we are ready to merge this I think.

**Comment by fidelram** [comment]
I want to give it a try before merging. Thanks a lot of the clean up, was badly needed.

**Comment by bgruening** [comment]
Thanks for the merge @fidelram!

**Pull request merged**